tests/topotests/bgp_snmp_bgp4v2mib/test_bgp_snmp_bgp4v2mib.py

```diff
@@ -308,7 +308,7 @@ def __get_notif_bgp4v2_in_trap_file(router):
     r2.vtysh_cmd("conf\nbgp snmp traps bgp4-mibv2")
     r2.vtysh_cmd("conf\nno bgp snmp traps rfc4273")
     rr.vtysh_cmd("clear bgp *")
-    sleep(30)
+    sleep(5)
```

Comment:
why not a run_and_expect a check to see peering is back after the clear?

Reply:
> why not a run_and_expect a check to see peering is back after the clear?

Actually there is a run and expect afterwards. I've only reduced the time to speed things up. But without the sleep the first run_and_expect will fail anyway. Just a short delay seems to be more optimal before starting the run_and_expect

Reply:
NAK -> This is just moving things around and *praying* that the sleep of 5 seconds is enough for reconvergence, especially on loaded systems..  The next run_and_expect is running snmp commands, not *whether* or not bgp has converged.  That was what the original sleep was for an attempt to give the tests enough time to let convergence happen.  WE HAVE FIXED too many cases where the tests make assumptions about state and make it incredibly hard to figure out what has gone wrong.   WHY are we perpetuating it?

Reply:
Ok, removed it. No idea why the original author added the sleep before the `run_and_expect`. Just wanted to lower the delay a bit. Hope this still works, but the initial check WILL go wrong, so now we have a sleep 10 instead of the sleep 5 (because that's the delay specified in the run_and_expect here)

Reply:
Force pushed new changes to address the wait intervals. Should hopefully address voiced concerns by involved parties so we can get this merged.